frontend/sw360-portlet/src/main/webapp/html/search/view.jsp

```diff
@@ -153,7 +153,17 @@
         result.push({
             "DT_RowId": '${doc.id}',
             "0": '${doc.type}',
-            "1": "<sw360:DisplaySearchResultLink searchResult="${doc}"/>"
+            <core_rt:choose>
+                <core_rt:when test="${doc.type.equals('project')
+                                   || doc.type.equals('component')
+                                   || doc.type.equals('release')
```

Comment:
Well, there is a longer story behind these (admittedly not the most beautiful) lines: there was a very nice solution with a wrapper tag deciding whether to use the OutTag or the DisplaySearchResultLink Tag. It worked every second time very nicely, and every second time it failed for some obscure reason connected with input streams. 

This was the quicker solution. We tried to use such a method as you suggest in the DisplayLinkToSearchResult Tag, But also this was not successful. Can you explain how to use a custom tld function? Just calling a function from the tag doesn't work since we have not instance of the Tag.